File: 240116/시험 통과 여부 확인하기 2/verify-test-passed-2.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int n;
    int cnt=0;
    scanf("%d",&n);
    for (int i=1;i<=n;i++){
        int arr[4];
        int sum=0;
        for (int j=0;j<4;j++){
            scanf("%d",&arr[j]);
            sum+=arr[j];
        }
        if ((double)sum/4>=60){
            printf("pass\n");
            cnt+=1;
        }
        else{
            printf("fail\n");
        }
    }
    printf("%d",cnt);
    return 0;
}

File: 240313/시험 통과 여부 확인하기 2/verify-test-passed-2.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int sum=0;
        int cnt=0;
        int[] arr=new int[5];
        for (int i=0;i<n;i++){
            for (int j=0;j<4;j++){
                arr[j]=sc.nextInt();
                sum+=arr[j];
            }
            if (sum>=240){
                System.out.print("pass\n");cnt+=1;
            }
            else{
                System.out.print("fail\n");
            }
            
            sum=0;
        }
        System.out.printf("%d",cnt);
    }
}